app/dashboard/src/layouts/Settings/AriaInput.tsx

```diff
@@ -11,6 +11,11 @@ import { tv } from '#/utilities/tailwindVariants'
 
 const SETTINGS_INPUT_STYLES = tv({
   extend: INPUT_STYLES,
+  variants: {
+    readOnly: {
+      true: { base: 'opacity-100', textArea: 'border-transparent' },
```

Comment:
I think the inputs in settings should be somehow hightlighted, now it's unclear that we can click on the name and edit it, since nothing is making it different from an ordinary text element